.drone.yml

```diff
@@ -1,13 +1,11 @@
-matrix:
-  NAME:
-    - ukhomeofficedigital/cdp-ci
-  DOCKER_REPO:
-    - quay.io
 pipeline:
   lint_troposphere:
-    image: quay.io/ukhomeofficedigital/cdp-ci:latest
+    image: python:3.7-alpine
     commands:
-      - source /troposphere/bin/activate
+      - apk add --no-cache gcc musl-dev make
+      - python3 -m venv venv
```

Comment:
In another ticket we could add python3 to cdp-ci.

Reply:
Maybe although, Chris and I talked about this and thought it was best to have specialized containers rather swiss army knives, hence the current approach

Reply:
I'd argue swiss army knives are actually nicer for CI. (So you only pull one image and it is likely to already exist.) But meh.

It's a little ugly to run `apk add` all the time.

But this doesn't matter for now.